src/assets/scss/main.scss

```diff
@@ -11,4 +11,32 @@
 
 
 // Custom Layout styles
-@import './layout/layout';
\ No newline at end of file
+@import './layout/layout';
+
+$breakpoint-mobile: 500px;
+
+
+html, body {
+    @media (max-width: $breakpoint-mobile) {
+    
+    max-width: 100%;
+    overflow-x: hidden;
+
+    }
+}
+
+@media only screen and (max-width: $breakpoint-mobile){
+    .pf-c-page__header-brand{
+        padding-left:0px;
+        padding: 0.3rem;
+    }
+
+    .pf-c-page__header-brand-toggle{
+    margin: 0 !important;
```

Comment:
Please lint your code before submitting PR.

Why is `!important` necessary here?

Reply:
ok I will keep that in mind , 
@jennydaman  I was not able to override the css without using , !important  at that time , tbh it was the last  straw , anyways I am looking into it ..